.github/workflows/release.yml

```diff
@@ -9,9 +9,10 @@ on:
 jobs:
   info:
     runs-on: ubuntu-latest
-    # the commit message will look like: `release: v{version} (#{pr-number})`
-    if: "startsWith(github.event.head_commit.message, 'release: v')"
+    # the commit message will look like: `release: {crate-name} v{version} (#{pr-number})`
+    if: "startsWith(github.event.head_commit.message, 'release: ')"
```

Comment:
I really don't like parsing text, IMHO it's fragile unless it has a fixed format (JSON, XML, etc).

Reply:
Yeah, I'm working on an action to make that more robust, but until then...

Reply:
I'm ok to use it for now.